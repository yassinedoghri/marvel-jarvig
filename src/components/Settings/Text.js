import styled from "styled-components";

const Text = styled.p``;

export default Text;
