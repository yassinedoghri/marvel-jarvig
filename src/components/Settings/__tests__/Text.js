import React from "react";
import renderWithTheme from "testHelpers";
import Text from "components/Settings/Text";

describe("Settings Item Text", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(<Text>Hello World!</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
