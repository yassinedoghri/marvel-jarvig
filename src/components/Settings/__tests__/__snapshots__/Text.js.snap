// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings Item Text renders correctly 1`] = `
<p
  className=""
>
  Hello World!
</p>
`;
